packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -1214,21 +1254,10 @@ export class Cluster extends ClusterBase {
    * stock `CfnCluster`), this is `undefined`.
    * @attribute
    */
-  public get clusterOpenIdConnectIssuerUrl(): string {
+  public get openIdConnectIssuerUrl(): string {
     return this._clusterResource.attrOpenIdConnectIssuerUrl;
   }
 
-  /**
-   * If this cluster is kubectl-enabled, returns the OpenID Connect issuer.
-   * This is because the values is only be retrieved by the API and not exposed
-   * by CloudFormation. If this cluster is not kubectl-enabled (i.e. uses the
-   * stock `CfnCluster`), this is `undefined`.
-   * @attribute
-   */
-  public get clusterOpenIdConnectIssuer(): string {
```

Comment:
Why did you remove this? Also a breaking change.

Reply:
will put it back